tests/integration/fileTests.py

```diff
@@ -449,9 +449,9 @@ def test_file_generation(self):
 
         self.assertEqual(response.status_code, 200)
         json = response.json
-        self.assertEqual(json["status"], "waiting")
+        self.assertIn(json["status"], ["waiting","finished"])
         self.assertEqual(json["file_type"], "D1")
-        self.assertIn("d1_data.csv", json["url"])
+        self.assertEqual("#", json["url"])
```

Comment:
Ah, thanks for clarifying that. Do we have some kind of test for the path where an actual D file comes back?